.github/workflows/coverage-report.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const fs = require('fs');
-            const commentBody = '```' + fs.readFileSync('coverage.log', 'utf8') + '```';
+            const commentBody = fs.readFileSync('coverage.log', 'utf8');
```

Comment:
I think the issue you're running into is that this change won't show up till this PR lands, because it is using `pull_request_target`

Reply:
The code fence was just my hacky way of dealing with the whitespace collapsing on the original format, but your other changes should address that